e2store/README.md

```diff
@@ -0,0 +1,16 @@
+# E2store
+
+E2store is a format originally developed by Nimbus as a framework for building other storage formats. More information can be found here https://github.com/status-im/nimbus-eth2/blob/stable/docs/e2store.md
+
+
+# What formats does this Crate support
+- e2store
+- era
+- era1
+
+## What is era?
+era is a format for storing beacon chain data more information can be found here https://github.com/status-im/nimbus-eth2/blob/stable/docs/e2store.md#era-files
+
+## What is era1
+
+era1 is a format for storing all of Ethereum's post merge blocks. It contains block headers, block bodies, and receipts for block 0-15537394
```

Comment:
nit: Should this be _pre-merge_?

Reply:
I meant in the first sentence:

_era1 is a format for storing all of Ethereum's **post merge** blocks_

Should this be "post merge" or "pre merge"?